.rubocop.yml

```diff
@@ -2,6 +2,7 @@ inherit_from: .rubocop_todo.yml
 
 AllCops:
   NewCops: enable
+  TargetRubyVersion: 2.6
```

Comment:
Do we need it? 
If I remember correctly, it should use the current ruby version when not specified

Reply:
If not specified it will check for the lowest `required_ruby_version` if specified in the Gemspec I think. I can remove it we don't really need it as it's already specified

Reply:
Yes, remove it